<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lmfaoown</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>413</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>631</width>
    <height>413</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>631</width>
    <height>413</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lmfaoown's Aion Hack</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar, QMenu {
	color: rgb(176, 30, 58);
	background-color: rgb(38, 38, 38);
}

QMenuBar::item:selected, QMenu::item::selected {
	background-color: rgb(50, 50, 50);
}

QMenu::item:selected {
	color: rgb(255, 255, 255);
	background-color: rgb(50, 50, 50);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="MainWindowStackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>636</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="mainPage_StackedWidget">
     <widget class="QLabel" name="mainPage_LoadingGif">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWebEngineView" name="webEngineView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>391</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>https://www.lmfaoown.com/aion_hack/launcher/</string>
       </url>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="aion_StackedWidget">
     <widget class="QLabel" name="aion_LoadingGif">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>353</y>
        <width>624</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {	
	color: rgb(176, 30, 58);
}</string>
      </property>
      <property name="text">
       <string>Receiving data from server...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="attach_StackedWidget">
     <widget class="QTableWidget" name="attach_TableWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>254</width>
        <height>326</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>126</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="4" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="4" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="5" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="6" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="6" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="7" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="7" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="8" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="8" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="9" column="0">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="9" column="1">
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="hack_PushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Hack</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_PushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="processId_TextEdit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>340</y>
        <width>94</width>
        <height>23</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="lineWrapColumnOrWidth">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>Insert process ID</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="home_MenuAction"/>
    <addaction name="hack_MenuAction"/>
    <addaction name="attach_MenuAction"/>
    <addaction name="separator"/>
    <addaction name="exit_MenuAction"/>
   </widget>
   <addaction name="menuNew"/>
  </widget>
  <action name="exit_MenuAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="home_MenuAction">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="hack_MenuAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hack (disabled)</string>
   </property>
  </action>
  <action name="attach_MenuAction">
   <property name="text">
    <string>Attach</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="7" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
